vendors/twtg/profiles/twtg_RFGroup1_1.0.4a_RP2-1.0.3_classA.yaml

```diff
@@ -0,0 +1,98 @@
+# ID that represents the profile of your device model.
+# SHALL Follow this convention: <vendorID> (lowercase) + '_' + <Profile Group> + '_' <LoRaWAN L2 & PHY versions> + '_class' + <LoRaWAN class A, B or C>
+# Follow the information below to know the profile group to use in the ID
+#    Profile Group
+#    RFGroup1:  eu863-870, kr920-923, in865-867, ru864-870
+#    RFGroup2:  us902-928, cn470-510, au915-928 (only for LoRaWAN 1.0.2revB or below)
+#    RFGroup3:   au915-928 (starting from LoRaWAN 1.0.2revC)
+#    RFGroup4: as923
```

Comment:
In general, why the profiles need to be in groups? Can't we make one profile with all regions in one?

Reply:
Hi Martin,
We must divide the ISM bands.
This is because each ISM band might have different LoRaWAN capabilities, but some are similar and even identical to some other, hence the need for RFGroups.
We do this because we need to let the user select which ISM band(s) the device needs to be configured on.